server/views/temporary-accommodation/lost-beds/new.njk

```diff
@@ -0,0 +1,29 @@
+{% extends "../../partials/layout.njk" %}
+{% from "../../partials/showErrorSummary.njk" import showErrorSummary %}
+{% from "../../partials/breadCrumb.njk" import breadCrumb %}
+{% set pageTitle = applicationName + " - Void bedspace" %}
+{% set mainClasses = "app-container govuk-body" %}
+{% block content %}
+    {{ breadCrumb('Record void bedspace', [
+    {title: 'List of properties', href: paths.premises.index()},
+    {title: 'View a property', href: paths.premises.show({ premisesId: premises.id })},
+    {title: 'View a bedspace', href: paths.premises.bedspaces.show({ premisesId: premises.id, roomId: room.id })}
+  ]) }}
+    {% include "../../_messages.njk" %}
+    <h1>Void a bedspace &nbsp; | &nbsp; {{ room.name }}</h1>
+    {{ showErrorSummary(errorSummary) }}
+    <div class="location-header">
+        <h2>Bedspace reference</h2>
+        <p>{{ room.name }}</p>
+        <h2>Property address</h2>
+        <p>{{ premises.addressLine1 }}<br/>{{ premises.postcode }}</p>
+    </div>
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <form action="{{ paths.lostBeds.create({ premisesId: premises.id, roomId: room.id }) }}" method="post">
+                <input type="hidden" name="_csrf" value="{{ csrfToken }}"/>
+                <input type="hidden" name="bedId" value="{{ room.id }}"/> {% include "./_editable.njk" %}
```

Comment:
Is this meant to be `value="{{ room.beds[0].id }}"`?

So I understand this correctly, this is so that in the create endpoint, we don't have to fetch the room again *just* to get the bed ID (this is what BookingsController does, but I think your solution here is good)

Reply:
ah yes good spot!

Yes, that and to maintain consistency in handling the form data. I think it was @richpjames idea actually, so props to him...